src/main/java/net/glowstone/block/blocktype/BlockLiquid.java

```diff
@@ -246,6 +247,9 @@ public void updatePhysics(GlowBlock block) {
         if (!(Byte.compare(block.getState().getRawData(), isWater(block.getType()) || block.getBiome() == Biome.HELL ? STRENGTH_MIN_WATER : STRENGTH_MIN_LAVA) == 0) || block.getRelative(DOWN).getType() == Material.AIR) {
             calculateFlow(block);
         }
+        if (initalData != block.getState().getRawData()) {
```

Comment:
It was meant to stop times where a blocks physics wasn't being run again when blocks where changing around it so it would be a leftover water block (that shouldn't be there) but after some testing I have realized that the check was unnecessary as the physics was already updating when blocks around it changed.